wpilibcExamples/src/main/cpp/examples/RamseteCommand/include/Robot.h

```diff
@@ -24,7 +26,7 @@ class Robot : public frc::TimedRobot {
  private:
   // Have it null by default so that if testing teleop it
   // doesn't have undefined behavior and potentially crash.
```

Comment:
This comment isn't relevant now that it's an optional.

Reply:
The comment isn't irrelevant; it's empty for that precise reason.

Reply:
I think this way it's fine, but I'm fine with rewording "null" to "empty".

Reply:
But it's not "null".  It's empty.  And there's no UB concern now; because it's an optional, it's getting initialized to empty by the optional default constructor.